types/xmldom-reborn/index.d.ts

```diff
@@ -0,0 +1,50 @@
+// Type definitions for xmldom-reborn 1.0.5
```

Comment:
For a new definition you should only include the MAJOR.MINOR part of the package version in the header so that the types-publisher can properly version the types package.